examples/ts-vue3/README.md

```diff
@@ -0,0 +1,25 @@
+# ts-vue3
+
+This template should help get you started developing with Vue 3 in Vite.
+
+## Customize configuration
+
+See [Vite Configuration Reference](https://vitejs.dev/config/).
+
+## Project Setup
+
+```sh
+npm install
```

Comment:
what about using yarn here as well? just for consistency

Reply:
Agree, fixed https://github.com/hCaptcha/vue-hcaptcha/pull/86/commits/b68905105c83328d4f25a18c2741a3eb7a0891d5